README.md

```diff
@@ -101,19 +101,27 @@ npm install --save pouchdb-hoodie-store
 Attach this plugin to the `PouchDB` object:
 
 ```js
-var PouchDB = require('pouchdb')
-PouchDB.plugin(require('pouchdb-hoodie-store'))
+var Store = require('pouchdb-hoodie-store')
 ```
 
 #### In the browser
 
-Include this plugin after `pouchdb.js` in your HTML page:
+Include this plugin in your HTML page:
 
 ```html
-<script src="node_modules/pouchdb/dist/pouchdb.js"></script>
 <script src="node_modules/pouchdb-hoodie-store/dist/pouchdb-hoodie-store.js"></script>
 ```
 
+Since `pouchdb.js` is bundled by default into the plugin, there's no need to load separately.  
+If you want to load your own PouchDB, just add before loading the plugin, now it'll use your `pouchdb.js` 
+  
+```html
+<script src="node_modules/pouchdb/dist/pouchdb.js"></script>
+```
+
+Additionally we will dist a "without PouchDB" version `pouchdb-hoodie-store-without-pouchdb.js` which you  
+could use in that case.
```

Comment:
`could => should` maybe? If we're definitely saying that we have a version without a bundled pouchdb.js, as I believe we are, then we should push people toward loading the appropriate version.

Also I'd change `in that case`, something more like, `[which you (could/should) use...] if you specify your own pouchdb version`.